admin/apple-actions/index/class-export.php

```diff
@@ -171,5 +172,34 @@ private function fetch_content_settings() {
 		return apply_filters( 'apple_news_content_settings', $settings );
 	}
 
+	/**
+	 * Overrides settings if a theme is explicitly mapped to the section for this post.
+	 *
+	 * @since 1.2.3
+	 * @param Settings $settings
```

Comment:
This line is missing the @param type and both of these @params are missing the description.